Implement SET_GLOBAL_SHIFT cli command (#1919)
CloudCompare/CloudCompare: CHANGELOG.md

				multiple arguments in each line allowed
				handle quoted arguments
				commands after this one will run after all commands in the file have been processed

> 		- SET_GLOBAL_SHIFT -KEEP_ORIGIN {TRUE/FALSE} {x} {y} {z}

Reviewer: And it may be clearer to add it after the xyz coordinates?
`-SET_GLOBAL_SHIFT {x} {y} {z} -KEEP_ORIGIN`

> 		- SET_GLOBAL_SHIFT -KEEP_ORIGIN {TRUE/FALSE} {x} {y} {z}

Author: Yes i was thinking about it to have only a command param like i did in the apply trans. Tommorow i'll change this. The positioning change can also work if you think that it is better.

> 		- SET_GLOBAL_SHIFT {x} {y} {z} -KEEP_ORIG_FIXED
> 			- set global shift on all entities, if -KEEP_ORIG_FIXED set then apply transformation to keep it in the same place. (it will warn if coordinate transformation is too big.)
> 			- KEEP_ORIG_FIXED is optional, default value is FALSE

Reviewer: ```suggestion
		- SET_GLOBAL_SHIFT {x} {y} {z} -KEEP_ORIG_FIXED
			- set global shift on all entities
			- if -KEEP_ORIG_FIXED is set, apply a transformation to keep the origin at the same place (a warning will be issued if coordinate transformation is too big). KEEP_ORIG_FIXED is optional.
```
